sensor_msgs/PointCloud2 point_cloud
int32[] indices
string name
objects_tracker/BoundingBox bb
geometry_msgs/Point mass_center